utils/frame/remote-externalities/Cargo.toml

```
@@ -1,5 +1,5 @@
 [package]
-name = "remote-externalities"
+name = "remote-ext"
```

```suggestion
name = "frame-remote-ext"
```

Or something similar, but it should be more expressive.

CC @kianenigma

---

The other name I've reserved is `frame-remote-externalities`. I'll change to that, then.